src/console.py

```diff
@@ -43,19 +51,17 @@ def get_top_border(board):
 
 
 def prompt_action():
-    invalid = True
-    move = 0
-    while invalid:
-        message = "Where do you want to move the pawn?\n" + \
-            "Choose in list and type ENTER (UP: {}, RIGHT: {}, DOWN: {}, LEFT: {} EXIT: {})\n" \
-            .format(commands[Action.UP], commands[Action.RIGHT], commands[Action.DOWN], commands[Action.LEFT], commands[Action.EXIT])
-        action = prompt(message)
-        try:
-            move = int(action)
-            invalid = False
-        except ValueError:
-            invalid = True
-    return move
+    key = prompt("Where do you want to move the pawn?\nChoose in list and type ENTER " + get_action_list())
+    return get_action(key)
+
+
+def get_action_list():
+    message = "("
+    for action, key in commands.items():
+        message += action.name + ": " + key + ", "
+    message = message[:-2]
```

Comment:
Better, instead of this kind of string manipulation, prefer the usage of the  `string.join(array)` function.

Usage:

```py
', '.join([1, 2, 3])
# 1, 2, 3
```